api/internal/target/kusttarget.go

```diff
@@ -251,7 +252,12 @@ func (kt *KustTarget) configureExternalGenerators() ([]resmap.Generator, error)
 	if err != nil {
 		return nil, err
 	}
-	return kt.pLdr.LoadGenerators(kt.ldr, kt.validator, ra.ResMap())
+	m := ra.ResMap()
+	err = m.AnnotateAll(kioutil.PathAnnotation, kt.ldr.Root())
```

Comment:
LMK if I should use a different annotation, but this one felt like it made sense.

Reply:
makes sense to me too

Reply:
Sorry to be a stickler, but I don't think this use case matches the annotation's description: `PathAnnotation records the path to the file the Resource was read from` -- that's not what we're recording here

Reply:
I can make a new annotation then. How about `WorkingDirAnnotation`, with a value `internal.config.kubernetes.io/working-dir`?